cmd/clusterctl/client/config/providers_client.go

```diff
@@ -72,6 +72,7 @@ const (
 	K0smotronProviderName      = "k0sproject-k0smotron"
 	IonosCloudProviderName     = "ionoscloud-ionoscloud"
 	VultrProviderName          = "vultr-vultr"
```

Comment:
@chrischdi seems like the same case as Huawei?

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/11861#discussion_r1959347509

Reply:
cc @fabriziopandini

Reply:
Hi, so we don't use multiple GH orgs (and currently I believe there are no plans to change that), just the the single https://github.com/OpenNebula, but we'll do what you think is best, I can revert last 2 commits no problem :+1::slightly_smiling_face:

Reply:
Thx. Let's wait for Fabrizio's feedback

Reply:
Talked to Fabrizio you can drop the prefix